wapitiCore/attack/mod_cms.py

```diff
@@ -0,0 +1,79 @@
+import re
+import json
+from os.path import join as path_join
+
+from wapitiCore.net.web import Request
+from wapitiCore.attack.attack import Attack
+from wapitiCore.language.vulnerability import LOW_LEVEL, _
+from wapitiCore.definitions.fingerprint import NAME as TECHNO_DETECTED
+
+MSG_TECHNO_VERSIONED = _("{0} {1} detected")
+MSG_NO_WP = _("No WordPress Detected")
+class mod_cms(Attack):
```

Comment:
If it is specific to Wordpress you should rename to `mod_wordpress_plugins` or `mod_wordpress_enumeration` so it is more clear

Reply:
`mod_wpenum` would be short and meaningful I guess